deployment/operator/Makefile

```diff
@@ -94,16 +94,22 @@ endif
 endif
 
 .PHONY: bundle
-bundle: copy-crds kustomize operator-sdk ## Generate bundle manifests and metadata, then validate generated files.
+bundle: copy-crds kustomize operator-sdk kustomizations ## Generate bundle manifests and metadata, then validate generated files.
 	$(OPERATOR_SDK) generate kustomize manifests -q
-	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
+	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMAGE_TAG_BASE):v$(VERSION)
+	find config/manifests/bases -name nri-plugins-operator.clusterserviceversion.yaml | xargs -I '{}' \
+		sed -e "s|containerImage:.*|containerImage: $(IMAGE_TAG_BASE):v$(VERSION)|" -i '{}';
```

Comment:
nitpick: `$(IMAGE_TAG_BASE):v$(IMAGE_VERSION)` is used multiple times here and you have an unused definition of `IMG =  $(IMAGE_TAG_BASE):$(IMAGE_VERSION)`. Maybe we could update/fix that to `IMAGE = $(IMAGE_TAG_BASE):v$(IMAGE_VERSION) ` and use `$(IMAGE)` here. WDYT ?

Reply:
@klihub done